app/src/main/java/ch/epfl/sweng/studyup/questions/DisplayQuestionActivity.java

```diff
@@ -60,25 +62,30 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_display_question);
 
-        int answerNumber = 0;
-        boolean trueFalse = false;
-        String questionTitle = "";
-        String questionID = "";
 
-        Intent intent = getIntent();
-        if(!checkIntent(intent)) return;
+        int answerNumber;
+        boolean trueFalse;
+        String questionTitle;
+        String questionID;
 
+        if (MOCK_ENABLED) {
```

Comment:
Why ? There are tests that check if the progress bar is indeed displayed

Reply:
Because, if we don't do this. Travis gets a could not launch intent, because something is "constantly repainting the screen".